### PR TITLE
Update version numbers for release

### PR DESCRIPTION
**Pull request by dungeondaddy**

### CODE REVIEW

File: omni_blocks/__init__.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
-__version__ = '2.0.0'
+__version__ = '2.0.3'
```
Comment (dammitjim) [inline omni_blocks/__init__.py]: Is there a reason we've bumped 3 patch versions here?

**Comment by SemVerTsar** [comment]
I think we've dropped bump version, so that code could likely be removed

File: omni_blocks/__init__.py (hunk 1 above)
Comment (aabele) [reply omni_blocks/__init__.py]: To emphasize the fast progress i guess...

Comment (SemVerTsar) [reply omni_blocks/__init__.py]: Looks like it was tagged, but not updated in the `__init__`

Comment (dungeondaddy) [reply omni_blocks/__init__.py]: @dammitjim I was very tired on Friday and made a mess of the two patch versions I tagged previously, could ignore those though and go to 2.0.1 if that makes more sense?

Comment (dungeondaddy) [reply omni_blocks/__init__.py]: I only realised this needed updating once I tried to `pip install` it

**Pull request merged**